WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -407,6 +407,25 @@ private extension OrderDetailsViewController {
             self?.reloadSections()
         }
     }
+
+    func syncOrderAfterPaymentCollection(onCompletion: @escaping ()-> Void) {
```

Comment:
We don't need a full "pull to refresh", as far as I understand. So this method only refreshes the order and the order notes.